### PR TITLE
blobuploadcleanupworker: Add  BLOBUPLOAD_DELETION_DATE_THRESHOLD (PROJQUAY-2915)

### PR DESCRIPTION
**Pull request by bcaton85**
Added the BLOBUPLOAD_DELETION_DATE_THRESHOLD internal property to configure the TTL on orphaned blobs.

### CODE REVIEW

**Review by flavianmissi — DISMISSED** [review]
LGTM

File: workers/blobuploadcleanupworker/blobuploadcleanupworker.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 logger = logging.getLogger(__name__)
 
-DELETION_DATE_THRESHOLD = timedelta(days=2)
+DELETION_DATE_THRESHOLD = timedelta(hours=12)
```
Comment (kleesc) [inline workers/blobuploadcleanupworker/blobuploadcleanupworker.py:18]: Can we make this configurable instead. Something like `BLOB_UPLOAD_DELETION_DATE_THRESHOLD`.
This way we can keep the current value as default, but allow it to be shorter or longer for others.

**Review by kleesc — CHANGES_REQUESTED** [review]

File: workers/blobuploadcleanupworker/blobuploadcleanupworker.py (hunk 1 above)
Comment (bcaton85) [reply workers/blobuploadcleanupworker/blobuploadcleanupworker.py]: Added

**Review by kleesc — APPROVED** [review]
LGTM

**Pull request merged**